Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```
@@ -1666,9 +1666,20 @@ script:
     - description: Comma-separated list of endpoint IDs. Can be retrieved by running the core-get-endpoints command.
       name: endpoint_ids
       required: true
-    - description: Comma-separated list of shell commands to execute.
+    - description: Comma-separated list of shell commands to execute. Set the `is_raw_command` argument to `true` to prevent splitting by commas. (Useful when using `||`, `&&`, `;` separators for controlling the flow of multiple commands).
       name: commands
       required: true
+    - description: Whether to pass the command as-is. When false, the command is split by commas and sent as a list of commands, that are run independently.
+      name: is_raw_command
+      type: Boolean
+      required: false
+    - description: 'Type of shell command. Possible values: "powershell", "native".' 
```

```suggestion
    - description: Type of shell command. 
```
If the possible values are listed under  "predefined" then there is no reason to list it in the description.  When you generate the README, the values are automatically added.

---

Removed "Possible values ..." in commit 2905ac8d6e0777c88b5721e74b7eb4ac02605645